tests/modules/programs/delta/delta-without-git-integration.nix

```diff
@@ -0,0 +1,23 @@
+{ config, ... }:
+{
+  programs.delta = {
+    enable = true;
```

Comment:
I think this will need a custom package stub with a build script because of the use of a wrapper in the module.

Reply:
Done, I wonder why `config.lib.test.mkStubPackage` just creates an empty dir, instead of a dir + empty executable